soc/arm/st_stm32/stm32wl/Kconfig.series

```diff
@@ -0,0 +1,17 @@
+# STMicroelectronics STM32WL MCU series
+
+# Copyright (c) 2020 STMicroelectronics.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_STM32WLX
+	bool "STM32WLx Series MCU"
+	select ARM
+	select CPU_CORTEX_M4
+	select CPU_CORTEX_M_HAS_DWT
+	select CPU_HAS_FPU
```

Comment:
Has it?

Reply:
You are right this is the first STM34 CortexM 4 without FPU. Thanks